uint8 number_leaders
uint64 id

uint8 robot_index_from
uint8 robot_index_to

float32 object_position_x
float32 object_position_y

float32 object_size_x
float32 object_size_y

float32 target_x
float32 target_y